mintpy/multi_transect.py

```diff
@@ -393,7 +393,7 @@ def read_fault_coords(Fault_coord_file, Dp):
 
 
 #####################################################################
-def main(argv):
+def main(argv=None):
```

Comment:
Entrypoints get called with no arguments, so this needs to be optional. Seems to work fine with just this change.

Reply:
LGTM. `add_attribute.py` and `multi_transect.py` seem to be the only two scripts having this issue. Thanks!